docs/range/demo/range.md

```diff
@@ -22,13 +22,13 @@ const style = {
     marginBottom: '40px',
     marginTop: '40px'
 };
-ReactDOM.render((
+ReactDOM.render(
```

Comment:
maybe you can remove the parentheses in `docs/range/demo/accessibility.md` `docs/range/demo/marks.md` `docs/range/demo/basic.md` , too